### DISCUSSION TITLE
Dark particles

### DISCUSSION THREAD

**Discussion by carlhihat** [creativelifeform/three-nebula#154] [Q&A]
Hello! I just recently found three-nebula and nebula.exe, and I'm very impressed! :D

I'm currently using two emitters to simulate tire smoke and tire tracks behind a player controlled car. The smoke already looks great, but I'm having some issues with the tire tracks which I can't make a dark color without them not showing. 

Is it possible to emit dark particle sprites, if so: how? :)

**Comment by rohan-deshpande** [comment]
Hi, thanks for using the library!

Can I ask you to clarify what you mean here

> I'm having some issues with the tire tracks which I can't make a dark color without them not showing

I am not really following, perhaps if you've got a demo I can look at?

**Reply by carlhihat** [reply]
Sorry for the late reply.. But I encountered the same issue again for a different project, and I remembered that I had asked the question once already!

[I hope this gif captured in the desktop app explains what I mean by trying to emit black/dark particles](https://i.gyazo.com/427d226fc5bd0c6037b1c4273e3ab9b4.mp4)

**Comment by QuisMagni** [comment]
Hi, i have the same problem. If i do not use the color behavior at all, the particles will be red and the color behavior is not doing anything for dark colors.

I think it is because the brightness value of a texture pixel is used for the opacity value. Is there anything i can do to avoid that behavior?

**Comment by QuisMagni** [comment]
After digging into the source code i came up the the following solution:

`          {
            "type": "BodySprite",
            "properties": {
              "texture": "./img/textures/smoke.png",
              "isEnabled": true,
              "materialProperties": {
                "blending": "NormalBlending"
              }
            }
          },`

For the behaviour:

`
          {
            "type": "Color",
            "properties": {
              "colorA": "#ffffff",
              "colorB": "#DDDDDD",
              "life": null,
              "easing": "easeOutCubic"
            }
          },
`
![image](https://user-images.githubusercontent.com/1906635/192976781-827ec9b6-a265-44cd-905c-0c2125f3369d.png)
